### DISCUSSION TITLE
No includepath for windows.h

### DISCUSSION THREAD

**Discussion by henraso** [tauri-apps/tauri#7742] [Q&A]
I have set up a project using yarn and create-react-app. While trying to use "yarn tauri dev" it cannot build the application.
This is my error:
`error: failed to run custom build command for `vswhom-sys v0.1.2`

Caused by:
  process didn't exit successfully: `C:\Users\user\Documents\tauri\src-tauri\target\debug\build\vswhom-sys-297c61eca2690ef3\build-script-build` (exit code: 1)
  --- stdout
  cargo:rerun-if-changed=build.rs
  cargo:rerun-if-changed=ext/vswhom.cpp
  TARGET = Some("x86_64-pc-windows-msvc")
  OPT_LEVEL = Some("0")
  HOST = Some("x86_64-pc-windows-msvc")
  cargo:rerun-if-env-changed=CXX_x86_64-pc-windows-msvc
  CXX_x86_64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CXX_x86_64_pc_windows_msvc
  CXX_x86_64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CXX
  HOST_CXX = None
  cargo:rerun-if-env-changed=CXX
  CXX = Some("C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.37.32822\\bin\\Hostx64\\x64\\cl.exe")
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  DEBUG = Some("false")
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64-pc-windows-msvc
  CXXFLAGS_x86_64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CXXFLAGS_x86_64_pc_windows_msvc
  CXXFLAGS_x86_64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CXXFLAGS
  HOST_CXXFLAGS = None
  cargo:rerun-if-env-changed=CXXFLAGS
  CXXFLAGS = None
  running: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.37.32822\\bin\\Hostx64\\x64\\cl.exe" "-nologo" "-MD" "-Brepro" "-W4" "-FoC:\\Users\\ohenr\\Documents\\TAPI\\tauri\\src-tauri\\target\\debug\\build\\vswhom-sys-975e20f723c6a75a\\out\\ext/vswhom.o" "-c" "ext/vswhom.cpp"
  vswhom.cpp
  **ext/vswhom.cpp(16): fatal error C1034: windows.h: no include path set**
  exit code: 2

  --- stderr

  error occurred: Command "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.37.32822\\bin\\Hostx64\\x64\\cl.exe" "-nologo" "-MD" "-Brepro" "-W4" "-FoC:\\Users\\user\\Documents\\tauri\\src-tauri\\target\\debug\\build\\vswhom-sys-975e20f723c6a75a\\out\\ext/vswhom.o" "-c" "ext/vswhom.cpp" with args "cl.exe" did not execute successfully (status code exit code: 2).

warning: build failed, waiting for other jobs to finish...
error Command failed with exit code 4294967295.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 4294967295.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 101.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

It seems that it cannot find "windows.h", but I know it exists. Is there anywhere I can include it?

**Comment by diogobis** [comment]
Hi, any updates on this? I'm getting the same error

**Reply by EdwardJoke** [reply]
## When I run `npm run tauri dev`, I get: 

error: failed to run custom build command for `vswhom-sys v0.1.3`

Caused by:
  process didn't exit successfully: `C:\Kuai-Ying\snapshot\src-tauri\target\debug\build\vswhom-sys-a894b7846af786aa\build-script-build` (exit code: 1)
  --- stderr


  error occurred in cc-rs: command did not execute successfully (status code exit code: 2): "C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.44.35207\\bin\\HostX64\\x64\\cl.exe" "-nologo" "-MD" "-Brepro" "-W4" "/Zm2000" "-FoC:\\Kuai-Ying\\snapshot\\src-tauri\\target\\debug\\build\\vswhom-sys-f181dcbae5c434a2\\out\\71b29add3157f200-vswhom.o" "-c" "ext/vswhom.cpp"


warning: build failed, waiting for other jobs to finish...
